bindings/java/src/main/java/org/openucx/jucx/ucp/UcpRequest.java

```diff
@@ -21,6 +22,14 @@
 
     private long senderTag;
 
+    /**
+     * The only request that doesn't have automatic status update (no callback)
+     * so need to go into ucx to check it's real status
+     */
+    private boolean isCloseRequest = false;
```

Comment:
can we have CB on close as well, for consistency?

Reply:
Since there's no callback in ucp layer, we can't - since the user may not explicitly progress this request, but rely on the global progress thread, that just loops in progress.

Reply:
ok

Reply:
Will remove it after #6036